guide/getting-started/software.md

```diff
@@ -169,8 +169,12 @@ A [node]({{ '/guide/glossary/node/' | relative_url }}) is simply a computer runn
    layout = "float-right-desktop"
 %}
 
+Bitcoin mining is the act of confirming the next block in the blockchain by being the first to solve a computationally intense math problem. The successful miner receives a block reward and the fees for the transactions contained in the new block in bitcoin.
```

Comment:
This just explains what mining is, not what mining software is. Would be better to explain the nuances of the different types of software you can run on mining hardware - should also cross link that with the hardware overview section on miners.

Reply:
This paragraph was added to aid in making sense of the one about Ligtning in the same section.
Can remove if it's seen as unnecessary.

Reply:
I agree with both of you. I think if we were to add `mining` here, we probably would want to add content specific to mining `software`, since mining was already covered on a few other pages.

However, I don't think application builders need to know the details of mining software in order to build applications. Mining software, is something that should concern miners only, so my vote is to, take out this paragraph, but re-use it perhaps in the tech primer PR #415

Reply:
OK, will delete.

Reply:
Posted a suggestion below (keeping the text in a shortened version with a link to the tech primer).